### PR TITLE
[10.0][MIG] geoengine_partner

### PR DESCRIPTION
**Pull request by petrus-v**

### CODE REVIEW

**Comment by yvaucher** [comment]
Please base this on https://github.com/OCA/geospatial/pull/128 plus a review would be welcome

File: geoengine_partner/README.rst [hunk 1]
```diff
@@ -16,14 +18,19 @@ Take a look at the installation section in the description of the module
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/geospatial/issues>`_.
-In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/geospatial/issues/new?body=module:%20geoengine_partner%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
```
Comment (yvaucher) [inline geoengine_partner/README.rst:22]: s/project_repo/geospatial

File: geoengine_partner/__manifest__.py [hunk 2]
```diff
@@ -1,37 +1,21 @@
...
      'base_geoengine'
  ],
  'data': [
-     'geo_partner_view.xml'
+     'views/geo_partner_view.xml'
  ],
- 'installable': False,
+ 'installable': True,
  'application': True,
  'active': False,
```
Comment (yvaucher) [inline geoengine_partner/__manifest__.py:20]: travis raised this issue:

```
geoengine_partner/__manifest__.py:5: [C8103(manifest-deprecated-key), ] Deprecated key "active" in manifest file
```

**Review by yvaucher — CHANGES_REQUESTED** [review]
Please rebase this on the changes made in 9.0 with PR #128

File: geoengine_partner/views/geo_partner_view.xml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    <record id="geoengine_vector_layer_res_partner" model="geoengine.vector.layer">
+        <field name="geo_field_id"
+            ref="geoengine_partner.field_res_partner_geo_point"/>
+        <field name="name">Partner location</field>
+        <field eval="6" name="sequence"/>
+        <field name="view_id" ref="ir_ui_view_geo_partner"/>
+        <field name="geo_repr">basic</field>
+        <field eval="1" name="nb_class"/>
+        <field name="begin_color">#FF680A</field>
+        <field name="symbol_binary">iVBORw0KGgoAAAANSUhEUgAAACAAAAAfCAYAAACGVs+MAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJ
```
Comment (dreispt) [inline geoengine_partner/views/geo_partner_view.xml:50]: FYI is is possible to provide binary content as a relative path to the actual file in the repo.

**Comment by dreispt** [comment]
TraviCI build is failing.

**Pull request closed without merging**